scripts/funcotator/data_sources/cosmic/createSqliteCosmicDb.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Can we have some usage docs?

Reply:
Sure.  FYI, these aren't really intended to be used frequently - they should only need to be run when updating data sources.